daffodil-test/src/test/resources/org/apache/daffodil/layers/ais.tdml

```diff
@@ -38,6 +38,9 @@ required to pad the data payload to a 6-bit boundary. So it is 0 to 5.
 The *5C is a NMEA 0183 data integrity checksum, preceded by "*" - computed on 
 the whole message including the AIVDM, but excluding the "!"
 
+According to the wikipedia article on NMEA 0183, this checksum is just an XOR of the
+specific message bytes.
+
 Seems another role for layers is computing checksums.
```

Comment:
Park a JIRA ticket for updating this AIS layer to implement the checksum. 

Fix comment: remove word "Seems".